CHANGES.md

```diff
@@ -84,7 +84,7 @@
 ## Known Issues
 
 * Fixed X (Java/Python) ([BEAM-X](https://issues.apache.org/jira/browse/BEAM-X)).
-
+* Fixed error while writing multiple DeferredFrames to csv (Python) ([BEAM-12701](https://issues.apache.org/jira/browse/BEAM-12701)).
```

Comment:
This section looked odd to me (is it for known issues, or fixed issues?). It turns out our changelog template got mangled at some point! There's supposed to be a bugfixes section.

I added it back in https://github.com/apache/beam/pull/15516. Would you mind merging master or rebasing this PR, and moving this line to the "Bugfixes" section?

Reply:
Done!